cloudevents/sdk/event/base.py

```diff
@@ -20,61 +20,141 @@
 # TODO(slinkydeveloper) is this really needed?
 class EventGetterSetter(object):
 
+    # ce-specversion
     def CloudEventVersion(self) -> str:
         raise Exception("not implemented")
```

Comment:
Shouldn't this be called `CloudEventSpecversion`?

Reply:
I wish, but unfortunately users are already using CloudEventVersion

Reply:
Can we make breaking changes for major versions of this package?